gef.py

```diff
@@ -8348,6 +8349,10 @@ def do_invoke(self, argv):
         return
 
     def context_title(self, m):
+        # allow for not displaying a title line
+        if m is None:
```

Comment:
Better:
```suggestion
        if not m:
```

Reply:
No this won't work, we already have a `if not m` below my code for displaying a title line with nothing else

Reply:
This addition would make it so you can display no title line at all